.github/workflows/colcon_in_container.yaml

```diff
@@ -14,19 +15,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
-        ros_version: [humble, iron, rolling]
+        os: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04]
+        ros_version: [jazzy, humble, iron, rolling]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
       - uses: canonical/setup-lxd@v0.1.0
       - name: Install dependencies
         run: |
           sudo apt update
           sudo apt upgrade -y
-          python -m pip install --upgrade pip
+          sudo rm /usr/lib/python3.*/EXTERNALLY-MANAGED || true
```

Comment:
What is that for?

Reply:
You can no longer install pip packages on the system on 24.04: https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-every-time-i-use-pip-3

Reply:
I picked this solution because it's less invasive.